cosredux/science.py

```diff
@@ -208,3 +222,30 @@ def coadd_exposures(x1d_files, segm, outfile, bin=None):
     # Write
     coadd.write(outfile, overwrite=True)
     print("Wrote {:s}".format(outfile))
+
+
+def combinespectfiles(spfile_a, spfile_b, file_ab):
+    """ Coadd two spectra, and write output in a file
+
+    Parameters
+    ----------
+    spfile_a : str
+    spfile_b : str
+       files with spectra
+    file_ab : str
+       output file with combined spectra
```

Comment:
FITS ?

I see.  It probably depends on the extension of the filename.
You should probably note this.